hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java

```diff
@@ -178,19 +180,22 @@ public void testSerialization() {
 
     private static final class Mode {
 
-        private final String mode;
+        public final int mode;
+        public final String stringMode;
```

Comment:
consider renaming fields `mode` to `functionType` (as reference to `type` field in `DoubleBiFunction`) and `stringMode` to `functionName`